contracts/relayer/BaseRelayer.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity ^0.4.24;
+
+import "../lib/sig/ECDSA.sol";
+import "../apps/AragonApp.sol";
+import "../common/DepositableStorage.sol";
+
+
+contract BaseRelayer is AragonApp, DepositableStorage {
+    using ECDSA for bytes32;
+
+    bytes32 public constant OFF_CHAIN_RELAYER_SERVICE_ROLE = keccak256("OFF_CHAIN_RELAYER_SERVICE_ROLE");
```

Comment:
What's the rationale for having a role for this? The auth check is definitely adding a decent amount of gas overhead.

Reply:
Yea, I just added it because it feels more safe this way, but it may not be dangerous to accept relayed calls from any off-chain service. The only possible attack we could face is miners as relayers where they could set a high gas price for example. 

An optimization to using the ACL could be to keep a whitelisted list of off-chain services, and use the ACL to write/remove addresses from that list instead.